VideoLocker/src/main/java/org/edx/mobile/module/analytics/ISegmentImpl.java

```diff
@@ -44,6 +50,21 @@ public void setAppNameContext() {
             this.properties.put(Keys.CONTEXT, getAppNameContext());
         }
 
+        public void setCustomDimensions() {
```

Comment:
I still think the name isn't ambiguous, cuz, we are only setting custom dimensions in the function.

made private, with its neighboring 2 functions as well.